cli/tests/activate.bats

```diff
@@ -287,6 +287,13 @@ EOF
   refute_output --partial "sourcing profile.fish"
   refute_output --partial "sourcing profile.tcsh"
   refute_output --partial "sourcing profile.zsh"
+}
+
+# bats test_tags=activate,activate:hook,activate:hook:bash
+@test "bash: command activate runs profile scripts" {
+  project_setup
+  sed -i -e "s/^\[profile\]/${HELLO_PROFILE_SCRIPT//$'\n'/\\n}/" "$PROJECT_DIR/.flox/env/manifest.toml"
+  sed -i -e "s/^\[hook\]/${VARS_HOOK_SCRIPT//$'\n'/\\n}/" "$PROJECT_DIR/.flox/env/manifest.toml"
```

Comment:
would it make sense to split this out into a function to aid with changes?

Reply:
Yeah or even just commit the files rather than using sed. Don't want to spend time on that in this PR though